MekHQ/src/mekhq/campaign/mission/AtBScenario.java

```diff
@@ -1852,10 +1787,6 @@ protected void loadFieldsFromXmlNode(Node wn) throws ParseException {
                         Entity en = null;
                         try {
                             en = MekHqXmlUtil.getEntityFromXmlString(wn3);
-                            if (wn3.getAttributes().getNamedItem("deployment") != null) {
```

Comment:
I presume deployment round is now calculated each time rather than saved?

Reply:
It's loaded in a different place, I think part of the companion Megamek PR.